.gitignore

```diff
@@ -12,3 +12,4 @@ website/src/jest/docs
 website/src/jest/blog
 lerna-debug.log
 npm-debug.log*
+.haste_cache
```

Comment:
rm -rf all the haste cache files and remove this line. We don't use haste cache any more so this is a leftover from a previous Jest version :)